api/client/clientRouter.js

```diff
@@ -268,4 +275,138 @@ router.delete('/:id/notes/:id', async (req, res) => {
   }
 });
 
+/*
+Intake Form Router
+*/
+
+router.get('/:id/intake', (req, res) => {
+  Clients.findById(req.params.id)
+    .then((client) => {
+      const household = Household.findHouseholdbyId(client['household_id']);
+      const locations = Household.findLocationByHouseholdId(
+        client['household_id']
+      );
+      const gender = Clients.getGenderIdByClientId(req.params.id);
```

Comment:
The gender_id is already on the clients table(object in promise) so we should be passing in `client.id`.

This applies to race and ethnicity as well